docs/api/app.md

```diff
@@ -723,6 +723,13 @@ Returns `string` - User operating system's locale two-letter [ISO 3166](https://
 
 **Note:** When unable to detect locale country code, it returns empty string.
 
+### `app.getSystemLocale()`
+
+Returns `string` - The current system locale, fetched using Chromium's `l10n_util` library.
+Possible return values are documented [here](https://source.chromium.org/chromium/chromium/src/+/main:ui/base/l10n/l10n_util.cc).
```

Comment:
Also, looking at the docs for `app.getLocale`, it's not clear to me what it returns if you don't specify the `--lang` flag explicitly. Does it fall back to the system locale or just default to English? That probably would have been good to document before, but while we're adding `app.getSystemLocale` it would be good to clarify that default behavior of `app.getLocale`. If it returns the same value as `app.getSystemLocale`, it would probably be sufficient to note that fact and link to the `app.getSystemLocale` section.

Reply:
For reference, `app.getLocale` calls into [l10n_util::GetApplicationLocale](https://source.chromium.org/chromium/chromium/src/+/main:ui/base/l10n/l10n_util.cc;drc=4c33ea7ce9a4064087f939723c0e9713193cf54f;l=573) which is much more involved.
All I know for now is that it might not return the same value as `app.getSystemLocale` by default. On my Windows machine, it returns the same value as `app.getSystemLocale`, but it seems like it can fall back to `en-US` on [GetApplicationLocaleInternalMac](https://source.chromium.org/chromium/chromium/src/+/main:ui/base/l10n/l10n_util.cc;l=498;drc=4c33ea7ce9a4064087f939723c0e9713193cf54f).

Reply:
:+1: on @itsananderson's suggestion of finding a version-friendly way of documenting the values for this new function and for `app.getLocale()` but it's out-of-scope for this PR